vignettes/customization.Rmd

```diff
@@ -117,6 +117,18 @@ template:
       fg: "#B8BCC2"
 ```
 
+### Navbar colors
+
+You can also customize the color scheme of the navbar by using the [`type` and `bg` parameters](https://getbootstrap.com/docs/4.0/components/navbar/#color-schemes):
+
+```yaml
+navbar:
+  type: dark
+  bg: primary
+```
+
+will produce a navbar with a dark background of the primary color (defined by Bootstrap default, a Bootswatch theme, or bslib variable).
```

Comment:
Alternatively, you could use the `bg` parameter to make the navbar more visually prominent, e.g. ...